packages/components/src/components/mobile-dialog/mobile-dialog.tsx

```diff
@@ -20,25 +34,27 @@ const MobileDialog = props => {
         header_classname,
     } = props;
 
-    const footer_ref = React.useRef(false);
+    const footer_ref = React.useRef<HTMLDivElement>(null);
     const [footer_height, setHeight] = React.useState(0);
     React.useLayoutEffect(() => {
         if (footer_ref.current && !footer_height) {
             setHeight(footer_ref.current.offsetHeight);
         }
     }, [footer, footer_height]);
 
+    const portal_element = document.getElementById(portal_element_id) as HTMLDivElement;
```

Comment:
```suggestion
    const portal_element = document.getElementById(portal_element_id);
```
@likhith-deriv Please avoid using `as` :(

Reply:
I typecasted it to avoid these errors: 
<img width="814" alt="Screen Shot 2022-11-16 at 11 50 35 AM" src="https://user-images.githubusercontent.com/98398322/202119887-0cd10a47-4df7-4b3a-adc1-8a17d8f75a71.png">
<img width="814" alt="Screen Shot 2022-11-16 at 11 50 48 AM" src="https://user-images.githubusercontent.com/98398322/202119896-f1180746-6d1a-4e3e-9026-f411274536cc.png">